### PR TITLE
Preload multiple times to catch up tip close enough & release 0.26.3

### PR DESCRIPTION
**Pull request by dahlia**

### CODE REVIEW

**Review by limebell — COMMENTED** [review]
- It would be great to have a test for following up new tips during preloading.
- Can you leave 'FIXME's and 'TODO's as issue after this PR is merged?

File: Libplanet/EnumerableExtensions.cs [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
+
+                first = false;
+            }
+
+            if (first)
+            {
+                throw new InvalidOperationException("Sequence contains no elements.");
+            }
+
+            return greatestElement;
```
Comment (riemannulus) [inline Libplanet/EnumerableExtensions.cs:53]: Is it possible to `null` when `if (first || value.CompareTo(greatestValue) > 0)` is all false?

**Comment by dahlia** [comment]
@limebell @riemannulus I rebased commits on the latest 0.26-maintenance.  Could you take a look again?

File: Libplanet/EnumerableExtensions.cs (hunk 1 above)
Comment (dahlia) [reply Libplanet/EnumerableExtensions.cs:53]: The `greatestElement` must not be `null`, except for empty sequence, and that's why `InvalidOperationException` is thrown if `first` is still `true` after the below loop ends.

**Review by riemannulus — APPROVED** [review]

**Pull request merged**